### PR TITLE
MAINT: remove code-book and add download links

### PR DESCRIPTION
**Pull request by mmcky**
This PR

- [x] removes the code-book
- [x] converts existing `md` files to `ipynb` files
- [x] provides a new `notebooks` page to link to github to download the notebook files for each chapter. 
- [x] @mmcky to do a review and error check

### CODE REVIEW

**Comment by mmcky** [comment]
@DrDrij can you help make this table look better in this context?

https://6604f5338a0863359318839a--keen-wozniak-c08666.netlify.app/notebooks

I copied the code from https://dp.quantecon.org/code.html but i think it doesn't look great when it is just a single line.

File: website/notebooks.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        <br>
+
+        <section id="links" class="main">
+            <header class="major">
+                <h2>Notebooks</h2>
+            </header>
+            <center>
+                <div class="table-wrapper">
+                    <table class="alt" style="width:50%;">
```
Comment (DrDrij) [inline website/notebooks.html:32]: @mmcky So just replace here lines 32-39 with the reformatted table that I have pasted below.

I have done it this way incase there is some detailed workflow that I am missing — but that HTML should get that page looking like so:

image



```
                    <table class="alt" style="width:50%;">
                        <tbody>
                            <tr>
                                <td>Python</td>
                            </tr>
                            <tr>
                                <td><a href="https://github.com/QuantEcon/book-networks-public/tree/main/notebooks">Files on GitHub</a></td>
                            </tr>
                        </tbody>
                    </table>
```

Does that help?

**Comment by DrDrij** [comment]
Hey @mmcky I hope you don't mind I went ahead and just committed the change to the table. 

This lines I edited are here: https://github.com/QuantEcon/book-networks-public/pull/62/files#diff-1935b95de07be14d4bfa51e8b6d87a248e2194e7dc56a775ebbf0ac4cafef1d1R32

The update should visually look like this:
image


And the HTML markup just added the second cell onto a new table row so it went from:

```
                    <table class="alt" style="width:50%;">
                        <tbody>
                            <tr>
                                <td>Python</td><td><a href="https://github.com/QuantEcon/book-networks-public/tree/main/notebooks">Files on GitHub</a></td>
                            </tr>
                        </tbody>
                    </table>

```
And now is:

```
                    <table class="alt" style="width:50%;">
                        <tbody>
                            <tr>
                                <td>Python</td>
			    </tr>
			    <tr>
                                <td><a href="https://github.com/QuantEcon/book-networks-public/tree/main/notebooks">Files on GitHub</a></td>
                            </tr>
                        </tbody>
                    </table>

```


Hope this is what you were after!

**Comment by mmcky** [comment]
thanks @DrDrij

**Comment by mmcky** [comment]
- [x] update location from `notebooks` to `ipynb` but that link won't work until site is published (@mmcky check all links once new site is published)

**Comment by mmcky** [comment]
@jstac this PR removes the code-book and replaces it with a static link to the collection of notebooks for each chapter. I don't see any collection of code in the `book-networks` repo so the source of truth are these notebooks as I understand. 

https://github.com/QuantEcon/book-networks-public/tree/remove-code-book/ipynb

**Comment by jstac** [comment]
> @jstac this PR removes the code-book and replaces it with a static link to the collection of notebooks for each chapter. I don't see any collection of code in the `book-networks` repo so the source of truth are these notebooks as I understand.
> 
> https://github.com/QuantEcon/book-networks-public/tree/remove-code-book/ipynb

Thanks @mmcky.  I don't think there are any corresponding notebooks in the private repo.

**Comment by mmcky** [comment]
@jstac are you happy for this to be merged now? 

The notebook link on 

https://6621bc8f2253ce485588bfed--keen-wozniak-c08666.netlify.app/

won't work until this is merged as the assets are part of this PR.

**Comment by jstac** [comment]
@mmcky Perhaps we can go ahead and merge, so that all the links work, and then iterate if we need to?

Could you please fix any failing tests that you need to and then merge?

**Comment by mmcky** [comment]
thanks @jstac I will merge then review the page in full.

**Pull request merged**